js_modules/dagster-ui/packages/ui-core/src/instigation/TickMaterializationsTable.tsx

```diff
@@ -0,0 +1,239 @@
+  HeaderCell,
+  Icon,
+  Inner,
+  Row,
+  RowCell,
+  Spinner,
+  Subtitle2,
+  TextInput,
+} from '@dagster-io/ui-components';
+import {useVirtualizer} from '@tanstack/react-virtual';
+import {useMemo, useRef, useState} from 'react';
+import {Link} from 'react-router-dom';
+import styled from 'styled-components';
+
+import {
+  AssetGroupAndLocationQuery,
+  AssetGroupAndLocationQueryVariables,
+} from './types/TickMaterializationsTable.types';
+import {gql, useQuery} from '../apollo-client';
+import {tokenForAssetKey} from '../asset-graph/Utils';
+import {AssetLink} from '../assets/AssetLink';
+import {AssetKeysDialogEmptyState} from '../assets/AutoMaterializePolicyPage/AssetKeysDialog';
+import {assetDetailsPathForKey} from '../assets/assetDetailsPathForKey';
+import {AssetDaemonTickFragment} from '../assets/auto-materialization/types/AssetDaemonTicksQuery.types';
+import {AssetKeyInput} from '../graphql/types';
+import {HeaderRow} from '../ui/VirtualizedTable';
+import {buildRepoAddress} from '../workspace/buildRepoAddress';
+import {workspacePathFromAddress} from '../workspace/workspacePath';
+
+const TEMPLATE_COLUMNS = '30% 17% 53%';
+
+export const TickMaterializationsTable = ({
+  tick,
+}: {
+  tick: Pick<
+    AssetDaemonTickFragment,
+    'requestedAssetKeys' | 'requestedMaterializationsForAssets' | 'autoMaterializeAssetEvaluationId'
+  > | null;
+}) => {
+  const [queryString, setQueryString] = useState('');
+
+  const filteredAssetKeys = useMemo(
+    () =>
+      tick
+        ? tick.requestedAssetKeys.filter((assetKey) =>
+            assetKey.path.join('/').includes(queryString),
+          )
+        : [],
+    [tick, queryString],
+  );
+
+  const parentRef = useRef<HTMLDivElement | null>(null);
+  const rowVirtualizer = useVirtualizer({
+    count: filteredAssetKeys.length,
+    getScrollElement: () => parentRef.current,
+    estimateSize: () => 34,
+    overscan: 10,
+  });
+  const totalHeight = rowVirtualizer.getTotalSize();
+  const items = rowVirtualizer.getVirtualItems();
+
+  const assetKeyToPartitionsMap = useMemo(() => {
+    const map: Record<string, string[]> = {};
+    tick?.requestedMaterializationsForAssets.forEach(({assetKey, partitionKeys}) => {
+      map[tokenForAssetKey(assetKey)] = partitionKeys;
+    });
+    return map;
+  }, [tick?.requestedMaterializationsForAssets]);
+
+  const content = () => {
+    if (queryString && !filteredAssetKeys.length) {
+      return (
+        <AssetKeysDialogEmptyState
+          title="No matching asset keys"
+          description={
+            <>
+              No matching asset keys for <strong>{queryString}</strong>
+            </>
+          }
+        />
+      );
+    }
+    if (!tick?.requestedAssetKeys.length) {
+      return (
+        <Box padding={{vertical: 12, horizontal: 24}}>
+          <Caption color={Colors.textLight()}>None</Caption>
+        </Box>
+      );
+    }
+    return (
+      <div style={{overflow: 'scroll', flex: 1}} ref={parentRef}>
```

Comment:
Use `Container` from our `VirtualizedTable` component set?

Reply:
Ahh good call, confirmed that this looks ok with the height: 100% on there, will switch it!

<img width="1585" alt="image" src="https://github.com/user-attachments/assets/fce0a50a-4517-45d8-bf09-39ad940b3b04">